datamodel.sv-display-mgmt.xml

```diff
@@ -14,7 +14,9 @@
 					</attributes>
 				</naming>
 				<display_template/>
-				<icon>images/display.png</icon>
+				<style>
+					<icon>asset/img/icons8-display.svg</icon>
+				</style>
```

Comment:
I guess this also needs a `itop_design` version update? To keep backward compatibility, I would just do:
```suggestion
				<icon>assets/img/icons8-display.svg</icon>
```